docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Write a Runtime Module as a Rust crate"
+---
+
+In this tutorial, you'll write a Substrate runtime module that lives in its own crate, and include it in a node based on the node-template. If you prefer you can follow along with the [video walkthrough](todo record after initial feedback).
+
+## Setup a Development Environment
+If you haven't already, follow the guide to [install substrate](getting-started/installing-substrate.md). You'll need the substrate-up scripts, but not the standard substrate node, so the `--fast` option is what you want here.
```

Comment:
Why not just add the one-liner here?

Reply:
For the same reasons we discussed in https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/issues/123 . The install process could (and hopefully will eventually) change. This way we don't have to update them in every single tutorial, but rather in a single central place.